CMakeLists.txt

```diff
@@ -18,6 +18,16 @@ find_package(ament_cmake_ros REQUIRED)
 
 ament_python_install_package(${PROJECT_NAME})
 
+if(UNIX)
+  include(cmake/rcutils_check_c_compiler_uses_glibc.cmake)
+  rcutils_check_c_compiler_uses_glibc(USES_GLIBC)
```

Comment:
One minor suggestion here: rename this to `check_c_compiler_uses_glibc()` (and the filename too).  I suspect that we'll be moving this bit of CMake functionality to a more generic place in the near future, so we may as well make it so we don't have to "break the API" here.

Reply:
I shied away from using generic naming, but I like it the most.